generators/client/templates/angular/src/main/webapp/app/shared/login/login.component.ts.ejs

```diff
@@ -71,7 +71,7 @@ export class <%=jhiPrefixCapitalized%>LoginModalComponent implements AfterViewIn
         }).then(() => {
             this.authenticationError = false;
             this.activeModal.dismiss('login success');
-            if (this.router.url === '/account/register' || this.router.url.startsWith('/account/activate/') ||
+            if (this.router.url === '/account/register' || this.router.url.startsWith('/account/activate') ||
 this.router.url.startsWith('/account/reset/')) {
```

Comment:
Is the trailing slash for `reset/` relevant here?

Reply:
In password reset router url is: `/account/reset/finish?key=....` - so `/account/reset/` is working fine in password reset.